kubernetes/loculus/templates/_common-metadata.tpl

```diff
@@ -75,6 +75,7 @@ fields:
     autocomplete: true
     displayName: Date released (exact)
     columnWidth: 100
+    # TODO change to be done here - only include these fields if a flag is set 
```

Comment:
tests fail because this isn't done yet and SILO expects the field to be there but it isn't.